symplyphysics/core/symbols/symbols.py

```diff
@@ -27,24 +27,25 @@ def display_name(self) -> str:
         return self._display_name
 
 
-class DimensionSymbolNew:
+class DimensionSymbolNew:  # pylint: disable=too-few-public-methods
```

Comment:
This class now looks rather useless. Did you try to remove it and use `Dimension` instead?

Reply:
But inheriting would imply an `is a` relationship but `DimensionSymbolNew` is not a `Dimension`, it just has a corresponding field.

Reply:
But it only has on field, which is Dimension. So basically, **it is** a Dimension.